src/prefect/cli/orion.py

```diff
@@ -150,6 +152,41 @@ async def start(
     app.console.print("Orion stopped!")
 
 
+@orion_app.command()
+async def status(
+    wait: bool = typer.Option(
+        False,
+        "--wait",
+        help="Runs health check loop until successful or optional timeout value is exceeded.",
+    ),
+    timeout: int = typer.Option(
+        None, "--timeout", help="Optional Timeout value in seconds"
+    ),
+):
+    """Verify Connection status with Orion API"""
+    sleep_time = 1  # Initial sleep time when polling
+    async with get_client() as client:
+        with (
+            anyio.move_on_after(timeout) if timeout else contextlib.nullcontext()
+        ) as timeout_scope:
+            while True:
+                response = await client.api_healthcheck()
+                if not response:
+                    break
+                else:
+                    if not wait:
+                        exit_with_error(response)
```

Comment:
What kind of responses do we typically see?

Reply:
The Health Check either returns None if successful or an Exception - All connection Attempts failed

Reply:
Is there anything specific that would be useful to do with the response I'm getting back here?